Apps/AdminWebClient/src/Server/Services/CsvExportService.cs

```diff
@@ -17,46 +17,80 @@ namespace HealthGateway.Admin.Services
 {
     using System;
     using System.Collections.Generic;
+    using System.Globalization;
     using System.IO;
+    using CsvHelper;
+    using CsvHelper.Configuration;
+    using HealthGateway.Admin.Server.Mappers;
     using HealthGateway.Common.Models;
+    using HealthGateway.Database.Delegates;
     using HealthGateway.Database.Models;
+    using HealthGateway.Database.Wrapper;
     using Microsoft.Extensions.Logging;
-    using ServiceStack.Text;
 
     /// <inheritdoc />
     public class CsvExportService : ICsvExportService
     {
-        private readonly ILogger logger;
+        private const int PageSize = 100000;
+        private const int Page = 0;
+        private readonly INoteDelegate noteDelegate;
+        private readonly IUserProfileDelegate userProfileDelegate;
+        private readonly ICommentDelegate commentDelegate;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="CsvExportService"/> class.
         /// </summary>
-        /// <param name="logger">Injected Logger Provider.</param>
```

Comment:
NIT:  We should not be removing the logger and should in general be adding debug statements at least.

Reply:
+1